vtex-apps/product-highlights
# Add support for 'product-highlights'  customization via Site Editor

#### What problem is this solving?

Enable editing of 'product-highlights' component via site-editor

#### How to test it?

1. Enter the workspace;

2. You will see that the product shelf, for example, will have a Black Friday stamp;

3. This stamp was inserted through the 'product-highlight';

4. If you access the workspace site-editor, you will see that now all the options that were previously only available via code can be accessed and changed via the site-editor.

[Workspace](https://sdtio876--diageo.myvtex.com/admin/cms/site-editor)

#### Screenshots or example usage:

![print-selo](https://user-images.githubusercontent.com/80407814/120839396-45d30180-c53f-11eb-8145-31f283049b7f.png)
